packages/mask/src/plugins/Tips/SNSAdaptor/TipsEntranceDialog.tsx

```diff
@@ -59,232 +62,154 @@ const useStyles = makeStyles()((theme) => ({
+    },
+}))
 
-const WalletButton: FC<WalletButtonProps> = ({ step, onClick }) => {
-    const { classes } = useStyles()
-    if (step === Step.AddWallet) return null
-    return (
-        <div className={classes.btnContainer}>
-            <Button onClick={onClick} className={classes.walletBtn} size="small">
-                {step === Step.Wallets ? Step.AddWallet : Step.Wallets}
-            </Button>
-        </div>
-    )
-}
+const supportedNetworkIds = [NetworkPluginID.PLUGIN_EVM]
 
 export function TipsEntranceDialog({ open, onClose }: TipsEntranceDialogProps) {
     const t = useI18N()
     const { classes } = useStyles()
     const { Storage } = useWeb3State()
+
     const [showAlert, setShowAlert] = useState(true)
-    const [step, setStep] = useState<Step>(Step.Main)
-    const supportedNetworkIds = [NetworkPluginID.PLUGIN_EVM]
+    const [pendingDefault, setPendingDefault] = useState<string>()
+
     const { showSnackbar } = useCustomSnackbar()
-    const account = useAccount(NetworkPluginID.PLUGIN_EVM)
 
     const { value: currentPersonaIdentifier } = useAsyncRetry(() => {
         setShowAlert(true)
         return Services.Settings.getCurrentPersonaIdentifier()
     }, [open])
+
     const { value: currentPersona } = useAsyncRetry(() => {
         return Services.Identity.queryPersona(currentPersonaIdentifier!)
     }, [currentPersonaIdentifier])
-    const clickBack = () => {
-        if (step === Step.Main) {
-            onClose()
-        } else {
-            setStep(Step.Main)
-        }
-    }
-
-    const { value: kv, retry: retryKv } = useAsyncRetry(async () => {
-        if (!Storage || !currentPersonaIdentifier) return
-        const storage = Storage.createNextIDStorage(
-            currentPersonaIdentifier.publicKeyAsHex,
-            NextIDPlatform.NextID,
-            currentPersonaIdentifier.publicKeyAsHex,
-        )
-
-        return storage.get<BindingProof[]>(PluginId.Tips)
-    }, [currentPersonaIdentifier, Storage])
 
-    const { loading, value: proofRes, retry: retryProof } = useProvedWallets()
-    const bindingWallets = useTipsWalletsList(proofRes, currentPersona?.identifier.publicKeyAsHex, kv)
+    const { loading, value: proofRes } = useProvedWallets()
 
-    const refetch = () => {
-        retryProof()
-        retryKv()
-    }
-    const refresh = () => {
-        setStep(Step.Main)
-        refetch()
-    }
+    const { value: TipsSetting, retry: retrySetting } = useTipsSetting(currentPersonaIdentifier)
 
-    const [pendingDefault, setPendingDefault] = useState<string>()
-    const defaultAddress = bindingWallets.find((x) => x.isDefault)?.identity
+    // Sort By `last_checked_at`
+    const bindingWallets = useMemo(() => {
+        if (!proofRes?.length) return EMPTY_LIST
+        return sortBy(proofRes, (x) => -Number.parseInt(x.last_checked_at, 10)).map(
+            (wallet, index, list): BindingProof => ({
+                ...wallet,
+                rawIdx: list.length - index - 1,
```

Comment:
Looks like `rawIdx` only for `fallbackName`, let's just construct it in the initialization.
```suggestion
                fallbackName: `Wallet ${list.length - index - 1}`,
```

```tsx
                        <WalletItem
                            key={x.identity}
                            setAsDefault={setAsDefault}
                            fallbackName={`Wallet ${x.rawIdx}`}
                            address={x.identity}
                            isDefault={isSameAddress(defaultAddress, x.identity)}
                        />
```